scripts/SANS/isis_reducer.py

```diff
@@ -847,15 +847,15 @@ def is_based_on_event(self):
         if self.is_can():
             sample = self.get_can()
             try:
-                dummy_ws = mtd[can.loader.wksp_name + "_monitors"]
+                mtd[can.loader.wksp_name + "_monitors"]
```

Comment:
add a note that this should be done using something like `if can.loader.wksp_name + "_monitors" in mtd.getObjectNames():`